arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1858,6 +1858,250 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/topics/{topicName}/subscriptions/{subscriptionName}/rules": {
+      "get": {
+        "tags": [
+          "Rules"
+        ],
+        "operationId": "Rules_ListAll",
+        "description": "List all the rules",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
```

Comment:
Request parameters here should also have constraints like minLength/maxLength and a regex to match